lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -247,7 +247,7 @@ def check_watched_items( self ):
                 assert external_job_id not in ( None, 'None' ), '(%s/%s) Invalid job id' % ( galaxy_id_tag, external_job_id )
                 state = self.ds.job_status( external_job_id )
             except ( drmaa.InternalException, drmaa.InvalidJobException ) as e:
-                if isinstance( e , drmaa.InvalidJobException ):
+                if isinstance( e, drmaa.InvalidJobException ):
```

Comment:
Yep.  I'm fixing a few of the : ones as well, but just not dictionaries intentionally formatted into columns, etc.